tests/unit/test_configuration.py

```diff
@@ -7,19 +7,53 @@
 from pseudopeople.configuration import NO_NOISE, Keys, get_configuration
 from pseudopeople.configuration.generator import DEFAULT_NOISE_VALUES
 from pseudopeople.configuration.interface import get_config
-from pseudopeople.configuration.validator import (
-    ConfigurationError,
-    validate_noise_level_proportions,
-)
+from pseudopeople.configuration.validator import ConfigurationError
 from pseudopeople.entity_types import ColumnNoiseType, RowNoiseType
 from pseudopeople.noise_entities import NOISE_TYPES
 from pseudopeople.schema_entities import COLUMNS, DATASETS
 
-PROBABILITY_VALUE_LOGS = [
-    ("a", "must be floats or ints"),
-    (-0.01, "must be between 0 and 1"),
-    (1.01, "must be between 0 and 1"),
-]
+PROBABILITY_VALUE_LOGS = {
+    "column_noise": [
+        ("a", "probabilities must be floats or ints"),
+        (-0.01, "must be between 0 and 1"),
+        (1.01, "must be between 0 and 1"),
+    ],
+    "row_noise": [
+        (
+            {Keys.ROW_PROBABILITY: "a"},
+            {
+                False: "probabilities must be floats or ints",
+                True: "Invalid parameter 'row_probability' provided",
+            },
+        ),
+        (
+            {Keys.ROW_PROBABILITY: 1.01},
+            {
+                False: "must be between 0 and 1",
+                True: "Invalid parameter 'row_probability' provided",
+            },
+        ),
+        (
+            {
```

Comment:
Why are configs specific to guardian-based duplication here but not those for other column noise types? It seems like either both or neither should be. How are the column noise configs tested?